pipelines/h3c/rtp/v1/task_scripts/do_ADD_LIBRARIAN_SSINS.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+set -e
+
+# import common functions
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# Parameters are set in the configuration file. Here we define their positions,
+# which must be consisten wtih the config.
+# 1 - (raw) filename
+fn=$(inject_hh ${1})
+
+# get the integer portion of the JD
+jd=$(get_int_jd ${1})
+
+midfix="_SSINS_"
+prefix="${fn%.*}"${midfix}
+
+for suffix in data.h5 z_score.h5 mask.h5 flags.h5 match_events.yml; do
+  file=${prefix}${suffix}
```

Comment:
Also, do these strings come out the way you're expecting them? Without knowing what the output files from SSINS are called, it's hard for me to tell. You'll probably have to test it locally to make sure.

Reply:
I just tested this and it looks like it works, although my `PATH` is not set up to actually run all these commands and make sure it finds the scripts, etc. It looks like it will need to be executed from the same directory as the data.

Reply:
That sounds like the level of checking I was referring to, and is working as intended. Thanks!